opal/mca/common/ucx/common_ucx.c

```diff
@@ -34,6 +34,37 @@ static void opal_common_ucx_mem_release_cb(void *buf, size_t length,
     ucm_vm_munmap(buf, length);
 }
 
+OPAL_DECLSPEC void opal_common_ucx_mca_var_register(const mca_base_component_t *component)
+{
+    static int registered = 0;
+    static int hook_index;
+    if (!registered) {
+        mca_base_var_register("opal", "ucx", "*", "verbose",
```

Comment:
@hoopoepg I brought up this MCA var on the weekly webex today.  People weren't wild about the idea of having a `*` in the var name -- among other reasons, it means you'll have to escape this var name on the shell command line.  Can you do something else, perhaps to effect an end name of `ucx_verbose` (with no middle part)?

Reply:
right, will fix var name

Reply:
@jsquyres fixed